js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -224,7 +224,7 @@ export const UserForm = ({
         </FormGroup>
       )}
       {isUserProfileEnabled ? (
-        <UserProfileFields />
+        <UserProfileFields config={user?.userProfileMetadata!} />
```

Comment:
Since the model defines `userProfileMetadata` as optional, I would prefer if we don't use the non-null assertion operator, as it can lead to errors around nullability, even if in this case the response might always be defined.

Reply:
it must be defined at this point because `isUserProfileEnabled` is true